arduino/smartcar/smartcar.ino

```diff
@@ -75,9 +78,9 @@ SmartCar car(arduinoRuntime, control, gyroscope, leftOdometer, rightOdometer);
 void setup()
 {
     Serial.begin(9600);
-  //Example: 
-    // chose to connect to localhost or external
-    connectHost(true); //choosing to connect to localhost.
+    //choose to connect to localhost or external
+    //choose true to connect to localhost.
+    connectHost(false);
```

Comment:
Should we keep connectHost(true); as default?

Reply:
I would say that the local host make more sense when testing, since there is a lot of "pollution" in the external broker.
But at the end, we just need to switch from one to another :)